.github/workflows/ci.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
         run: |
           mvn -V -B -s .github/mvn-settings.xml validate -Pide,validate-format
   linux-build-released-jvm:
```

Comment:
Do we want to keep the `released` infix here (and in all the other jobs)?

Reply:
imho not needed, we need just the names shorter to see the details in checks section bellow